backend/apps/owasp/models/badge.py

```diff
@@ -0,0 +1,28 @@
+"""Badge model for user achievements and roles in OWASP Nest."""
+
+from django.core.validators import MinValueValidator
+from django.db import models
+
+from apps.common.models import TimestampedModel
+
+
+class Badge(TimestampedModel):
+    """Represents a user badge for roles or achievements."""
+
+    name = models.CharField(max_length=255, unique=True, help_text="Name of the badge.")
+    description = models.CharField(
+        max_length=255, blank=True, help_text="A short description of the badge."
+    )
+    weight = models.PositiveSmallIntegerField(
+        default=0, validators=[MinValueValidator(0)], help_text="A weight to sort badges by."
+    )
+    css_class = models.CharField(
+        max_length=255, help_text="The font-awesome css class for the badge."
+    )
```

Comment:
Please check our existing model definitions and follow the approach used everywhere else (the verbose names, default values, ordering), e.g. [project model](https://github.com/OWASP/Nest/blob/852e54b2058794a29276db7e96f3daee98c6711f/backend/apps/owasp/models/project.py#L26).